public/auto-updater/index.js

```diff
@@ -1,8 +1,8 @@
 const os = require('os');
-const fetch = require('node-fetch');
-const { dialog, app } = require('electron');
+const { dialog } = require('electron');
```

Comment:
Is this a auto-generated code or written by us? If written by us, would it be possible to make it follow codestyle we have for other files in the project (eslint, prettier)?

Reply:
This is our code. We could do what you suggest. Would it be better to do it in a separate PR, since it is a different issue?

Reply:
Yes sure